### PR TITLE
Make ISO support optional

### PR DESCRIPTION
**Pull request by sean797**

### CODE REVIEW

File: spec/classes/pulp_apache_spec.rb [hunk 1]
```diff
@@ -172,8 +172,28 @@
...
-Alias /pulp/isos /var/www/pub/https/isos
+# -- GPG Keys -------------------
+Alias /pulp/gpg /var/www/pub/gpg
+
+<Directory /var/www/pub/gpg/>
+    Options FollowSymLinks Indexes
+</Directory>
+')
+
+        is_expected.to contain_file('/etc/pulp/vhosts80/rpm.conf').with(
```
Comment (ekohl) [inline spec/classes/pulp_apache_spec.rb]: ```verify_exact_contents(catalogue, '/etc/pulp/vhosts80/rpm.conf', ['Alias /pulp/exports /var/www/pub/yum/http/exports'])``` should be equal and easier to read. There are more cases in this file that could benefit from it.

File: examples/pulp_isorepo.pp [hunk 2]
```diff
@@ -1,7 +1,5 @@
 class { '::pulp':
   manage_repo  => true,
-  # https://github.com/theforeman/puppet-pulp/issues/138
```
Comment (ekohl) [inline examples/pulp_isorepo.pp]: Isn't this still an issue?

**Comment by ekohl** [comment]
With the deb support merged, this needs a rebase now.

**Comment by sean797** [comment]
Thanks @ekohl updated

File: spec/classes/pulp_apache_spec.rb [hunk 3]
```diff
@@ -188,19 +207,10 @@
...
-Alias /pulp/gpg /var/www/pub/gpg
-
-<Directory /var/www/pub/gpg/>
-    Options FollowSymLinks Indexes
-</Directory>
 ')
 
-        is_expected.to contain_file('/etc/pulp/vhosts80/rpm.conf').with(
+        is_expected.to contain_file('/etc/pulp/vhosts80/iso.conf').with(
         :content => 'Alias /pulp/isos /var/www/pub/http/isos
```
Comment (ekohl) [inline spec/classes/pulp_apache_spec.rb]: It might be slightly easier to read if you use ```"Alias /pulp/isos /var/www/pub/http/isos\n"```

Comment (sean797) [reply spec/classes/pulp_apache_spec.rb]: agreed, thanks!

**Comment by ekohl** [comment]
Restarted the Travis build to see if it'll pass now or fails for unrelated reasons. Will merge it after.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**